examples/hello-node-express-ts/index.ts

```diff
@@ -20,6 +22,12 @@ app.get('/', async (_req: Request, res: Response, next: NextFunction) => {
     res.setHeader('Content-Type', 'text/plain');
     const sayHello = () => 'Hello world!';
     const tracer: Tracer = trace.getTracer('hello-world-tracer');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
Hmm... maybe for now we should instead add a file disable at the top instead of line-by-line so this looks less cluttered, with a note on what it's there for.

```js
/* eslint @typescript-eslint/no-unsafe-member-access: 0 */
// ignore errors on metrics potentially having "any" value
```

Reply:
Changed the eslint to `warn` like we had talked about instead!

Reply:
👍  love it